playground/src/context-menu-test.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
+import { useFloating } from '@floating-ui/vue';
+import { ref } from "vue";
+
+let showMenu = $ref<boolean>(false)
+
+const parent = ref<HTMLElement>();
+const child = ref<HTMLElement>();
+const { floatingStyles } = useFloating(parent, child, {placement: 'bottom-start'});
```

Comment:
I wouldn't recommend testing anything relating to floating UI here. The library itself could break or change, and maintenance would then would be complicated for someone trying to introduce an unrelated change to the library.

Reply:
what do you think of leaving the floating UI stuff in this test file as an example? I am not writing any playwright tests related to floating UI